### PR TITLE
Adding solutions for exercise two.

### PR DESCRIPTION
**Pull request by lewisgcm**

### CODE REVIEW

File: chapter_02/lewis/exercise_1.pl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+###
+# (C) 2017 Lewis Maitland
+# O'Reily Excersises
+###
+
+use strict;
+use warnings;
+use Math::Trig ':pi';
```
Comment (alexdeas) [inline chapter_02/lewis/exercise_1.pl:8]: Using modules comes up in a later chapter.

File: chapter_02/lewis/exercise_1.pl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+###
+# (C) 2017 Lewis Maitland
+# O'Reily Excersises
+###
+
+use strict;
+use warnings;
+use Math::Trig ':pi';
+
+my $radius = $#ARGV == 0 ? $ARGV[0] : 12.5;
```
Comment (alexdeas) [inline chapter_02/lewis/exercise_1.pl:10]: $#ARGV works, but probably not for the reason you think it does. If you want to check how many elements are in an array, use ``scalar``.

File: chapter_02/lewis/exercise_1.pl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+# O'Reily Excersises
+###
+
+use strict;
+use warnings;
+use Math::Trig ':pi';
+
+my $radius = $#ARGV == 0 ? $ARGV[0] : 12.5;
+
+if( $radius !~ /^[0-9]+(\.[0-9]+)?$/ ) {
```
Comment (alexdeas) [inline chapter_02/lewis/exercise_1.pl:12]: Regular expressions come up in a later chapter!

File: chapter_02/lewis/exercise_1.pl [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	print "Invalid radius supplied.\n";
+	exit 0;
```
Comment (alexdeas) [inline chapter_02/lewis/exercise_1.pl:14]: ``exit 0`` implies the script ended successfully, which the message doesn't seem to imply. If you want to exit a script unsuccessfully use ``die``. If you want to exit early but successfully, use ``exit`` without specifying an exit code. It's best to leave the exit codes alone unless you know what you're doing with them.

File: chapter_02/lewis/exercise_3.pl [hunk 5]
```diff
@@ -0,0 +1,28 @@
+###
+# (C) 2017 Lewis Maitland
+# O'Reily Excersises
+###
+
+use strict;
+use warnings;
+
+use constant TITLE => " MULTIPLICATOR v1.01 ";
```
Comment (alexdeas) [inline chapter_02/lewis/exercise_3.pl:9]: Probably don't need to use constants in scripts this simple. You're not the first java developer I've seen do this though.

File: chapter_02/lewis/exercise_1.pl (hunk 2 above)
Comment (jhhmastercard) [inline chapter_02/lewis/exercise_1.pl:10]: The 2 exercises were to use a hard-coded value and a user input.

File: chapter_02/lewis/exercise_1.pl (hunk 3 above)
Comment (jhhmastercard) [reply chapter_02/lewis/exercise_1.pl:12]: Or just \d

**Pull request merged**